Gemfile.lock

```diff
@@ -61,19 +63,18 @@ GEM
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.3.3)
-      actionpack (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
+    railties (6.1.3)
+      actionpack (= 6.1.3)
+      activesupport (= 6.1.3)
       method_source
       rake (>= 0.8.7)
-      thor (>= 0.20.3, < 2.0)
-    rake (13.0.1)
-    thor (1.0.1)
-    thread_safe (0.3.6)
-    tzinfo (1.2.7)
-      thread_safe (~> 0.1)
-    zeitwerk (2.4.0)
-    zxcvbn-ruby (1.1.0)
+      thor (~> 1.0)
+    rake (13.0.3)
```

Comment:
### [rake](https://github.com/ruby/rake)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 13.0.1 → 13.0.3

[_[change-log](https://github.com/ruby/rake/blob/v13.0.3/History.rdoc), [source-code](https://github.com/ruby/rake/tree/v13.0.3)_]





<details>
<summary>Commits</summary>

A change of **31** commits. See the full changes on [the compare page](https://github.com/ruby/rake/compare/v13.0.1...v13.0.3).

These are the first 10 commits:
- (c4a50da) [Test Rake on TruffleRuby](https://github.com/ruby/rake/commit/c4a50da6217eacaef26d0d34d2e11c551bfe54ba)
- (4a4bc2e) [Enable GitHub Actions on pull requests](https://github.com/ruby/rake/commit/4a4bc2ea071d7cb70d83505955a4d9b53f1e1e96)
- (18508c4) [Skip the only failing test on TruffleRuby 19.3.0](https://github.com/ruby/rake/commit/18508c4845b1c7026e645295f4ceb27a46308347)
- (8d3c549) [Merge pull request #331 from eregon/test-truffleruby](https://github.com/ruby/rake/commit/8d3c5492b86567d8915e0fbea8597582d1fad876)
- (1fec8e2) [Do not include `.github` directory into released gem](https://github.com/ruby/rake/commit/1fec8e27731fca0d86fa8cde8f0008cba03402ab)
- (3137211) [Merge pull request #333 from voxik/patch-1](https://github.com/ruby/rake/commit/313721164307fca6375239c636fc7cd8680f175b)
- (5dd9db0) [Improve version check in test_rake_clean.rb](https://github.com/ruby/rake/commit/5dd9db024e525177b34199d374fcffb3ca4aafb8)
- (9e83d30) [Use ruby/setup-ruby to simplify CI](https://github.com/ruby/rake/commit/9e83d30cdc8b442b51698221fb280c91a118488d)
- (1111437) [Merge pull request #335 from eregon/use-setup-ruby](https://github.com/ruby/rake/commit/1111437901e7217b3265aafe9333f1431d30fe50)
- (5dd198c) [Test with jruby-head](https://github.com/ruby/rake/commit/5dd198c76781d05c92cd6df18e0700a14a47aedc)


</details>